src/actions/emojis.js

```diff
@@ -94,17 +98,18 @@ export function getCustomEmojisByName(names) {
         const promises = [];
         names.forEach((name) => promises.push(getCustomEmojiByName(name)(dispatch, getState)));
 
-        return Promise.all(promises);
+        await Promise.all(promises);
+        return {data: true};
```

Comment:
Can you elaborate on what's changing here exactly -- I don't immediately follow.

Reply:
The reason is the expected return. of the function, defined in ActionFunc.